modules/mesaides/templates/nginx_config.erb

```diff
@@ -123,6 +123,14 @@ server {
   }
   <% else %>
   location / {
+
+    proxy_set_header X-Real-IP       $remote_addr;
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header Host            $http_host;
+    proxy_set_header Connection      "";
+
+    proxy_http_version 1.1;
+
```

Comment:
Well, actually, I copy-pasted from StackOverflow and from a config I have on one of my servers 😬
But this can be useful if we need to implement some kind of filtering based on IP I suppose.